src/components/AppBar.tsx

```diff
@@ -89,14 +59,45 @@ export default function AppBar() {
   }));
 
   return (
-    <AppBarMod elevation={0} position="absolute" open={open}>
+    <AppBarMod>
       <Toolbar
         sx={{
+          ml: 2,
           display: "flex",
           alignItems: "center",
           justifyContent: "space-between",
         }}
       >
+        <Box
+          sx={{
+            flexGrow: 0,
+            display: "flex",
+            alignItems: "flex-start",
+            ml: -2,
+          }}
+        >
+          <AccountTreeOutlinedIcon
+            sx={{
+              display: { xs: "none", md: "flex" },
+              mr: 2,
+            }}
+            color="secondary"
+          />
+          <Typography
+            component="h1"
+            variant="h4"
+            color="secondary"
+            noWrap
+            sx={{
+              mr: 2,
+              display: { xs: "none", md: "flex" },
+              fontFamily: "",
+              fontWeight: 400,
+            }}
+          >
+            SkillSync
+          </Typography>
+        </Box>
         <Search>
```

Comment:
I would actually delete the Search-Component completely, since fist it is not displayed in the Applicant-Page and second of all i don't see the point why we need it anyways when there is a search field on almost every page anyway.